Index: out/production/codingInterview_prep/com/PatkarSir_core/TrickyQuestions/UsefulLinks
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/out/production/codingInterview_prep/com/PatkarSir_core/TrickyQuestions/UsefulLinks b/out/production/codingInterview_prep/com/PatkarSir_core/TrickyQuestions/UsefulLinks
new file mode 100644
--- /dev/null	
+++ b/out/production/codingInterview_prep/com/PatkarSir_core/TrickyQuestions/UsefulLinks	
@@ -0,0 +1,1 @@
+1. Medium article  : https://levelup.gitconnected.com/tricky-java-interview-questions-cfc546fd03ab
\ No newline at end of file
Index: out/production/codingInterview_prep/com/PatkarSir_core/TrickyQuestions/UsefulLinks
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/out/production/codingInterview_prep/com/PatkarSir_core/TrickyQuestions/UsefulLinks b/out/production/codingInterview_prep/com/PatkarSir_core/TrickyQuestions/UsefulLinks
new file mode 100644
--- /dev/null	
+++ b/out/production/codingInterview_prep/com/PatkarSir_core/TrickyQuestions/UsefulLinks	
@@ -0,0 +1,1 @@
+1. Medium article  : https://levelup.gitconnected.com/tricky-java-interview-questions-cfc546fd03ab
\ No newline at end of file
